test/python/test_axpy_var_pipelined.py

```diff
@@ -0,0 +1,41 @@
+import sys
+import pytest
+import getpass
+import tensorflow as tf
+import ngraph_bridge
+
+import numpy as np
+from common import NgraphTest
+
+# This test needs access to axpy_pipelined.py script
+# present in ngraph-bridge/examples
+# so this path needs to be added to python path when running this
+
+# For eg. when running the test from ngraph-bridge/build_cmake/test/python
+# you can add this path as below
+#sys.path.insert(0, '../../examples')
+
+from axpy_var_pipelined import *
+
+
+class TestAxpyPipelined(NgraphTest):
+
+    def test_axpy_pipelined(self):
+        #prefetch_env = "NGRAPH_TF_USE_PREFETCH"
+        ngraph_backend_i = "NGRAPH_TF_BACKEND"
```

Comment:
dont need to set backend now as cpu will run through parallel exec

Reply:
Removed this env